python.cmake

```diff
@@ -394,6 +394,12 @@ MACRO(PYTHON_BUILD MODULE FILE)
   STRING(REGEX REPLACE "/|:" "_" FILE_TARGET_NAME
     "${CMAKE_CURRENT_BINARY_DIR}/${MODULE}/${FILE}c")
 
+  # Remove extra C:\* on Windows
+  IF(WIN32)
+    STRING(REGEX REPLACE "^[^:_]*:_" "" FILE_TARGET_NAME
+      "${FILE_TARGET_NAME}")
+  ENDIF(WIN32)
+
```

Comment:
If this issue is too troublesome, we can use add_custom_command instead.

Reply:
I agree. This would be the cleanest solution.

Reply:
@nim65s What do you think?

Reply:
I don't have any useful comment for code enclosed in `IF(WIN32)`.

Reply:
I was mentioning the suggestion made by @jmirabel to gather together all the generated Python files.

Reply:
Now I get it. Yes, that was clearly a good idea :)